openfisca_uk/parameters/benefits/working_tax_credit/means_test/CTC_only_threshold.yaml

```diff
@@ -0,0 +1,9 @@
+description: Yearly income threshold after which child tax credits are reduced for benefit units only claiming CTC and not WTC.
```

Comment:
Similar to above, would this be describable as an earnings disregard or personal allowance?

Reply:
I think an earnings disregard is probably clearest, though I have seen either used.